actions/remove.go

```diff
@@ -22,17 +22,16 @@ import (
 //RemoveCommand to remove all codewind and project images
 func RemoveCommand(c *cli.Context) {
 	tag := c.String("tag")
-	imageArr := [5]string{}
+	imageArr := [4]string{}
 	imageArr[0] = "eclipse/codewind-pfe"
 	imageArr[1] = "eclipse/codewind-performance"
 	imageArr[2] = "eclipse/codewind-initialize"
 	imageArr[3] = "cw-"
-	imageArr[4] = "appsody"
 	networkName := "codewind"
 
 	if tag != "" {
 		for i := 0; i < len(imageArr); i++ {
-			if i == 3 || i == 4 {
+			if i == 3 {
```

Comment:
Can we just change this whole section to:
```
    imageArr := []string{
        "eclipse/codewind-pfe-amd64:" + tag,
        "eclipse/codewind-performance-amd64:" + tag,
        "eclipse/codewind-initialize-amd64:" + tag,
        "cw-",
    }
```
and initialise the array directly rather than initialising it then walking over most of the elements?

Reply:
agreed